### ISSUE TITLE
Accessibility-> Site page breadcrumb has a link to home page with an unnamed icon

### ISSUE THREAD

**Issue by manuji** [bcgov/nr-rec-resources#662]
**Describe the bug**
On Site page, in the breadcrumbs, there is an icon linked to the home without a name. For users using the screen reader the icon is read as "icon" and no context given. This issue fails the [1.1.1 Non-text Content (level A)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content) criterion. 

**To Reproduce**
1. Go to any site page (eg: 'https://d1vrz2gt4nmfe5.cloudfront.net/resource/REC204117')
2. Using the screen reader check the breadcrumbs of the page
3. Check how the icon linked to Home page is read by the screen reader. 

**Expected behavior**
No text content should have proper labels, names or explanations that serves the equivalent purpose.   

**Screenshots**

<img width="922" alt="Image" src="https://github.com/user-attachments/assets/342c2dcd-a6ef-4d12-adf0-6b5615f0f568" />

**Additional context**
Tested using MacOS, Chrome and VoiceOver.
